code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -66,14 +66,15 @@
 			to_chat(user, "<span class='warning'>[target] is full.</span>")
 			return
 
-		var/refill = reagents.get_master_reagent_id()
+		//var/refill = reagents.get_master_reagent_id() //FULPSTATION FIXES by Surrealistik April 2020 This is a half-assed feature often mistaken as a bug that is utterly broken as hell. Good riddance
 		var/trans = src.reagents.trans_to(target, amount_per_transfer_from_this, transfered_by = user)
 		to_chat(user, "<span class='notice'>You transfer [trans] units of the solution to [target].</span>")
 
+		/* //FULPSTATION FIXES by Surrealistik April 2020 This is a half-assed feature often mistaken as a bug that is utterly broken as hell. Good riddance.
 		if(iscyborg(user)) //Cyborg modules that include drinks automatically refill themselves, but drain the borg's cell
 			var/mob/living/silicon/robot/bro = user
 			bro.cell.use(30)
-			addtimer(CALLBACK(reagents, /datum/reagents.proc/add_reagent, refill, trans), 600)
```

Comment:
with this being removed, how do borgs refill their drinks? manually? just curious, dont have much experience with them